### ISSUE TITLE
Can we get a new release please?

### ISSUE THREAD

**Issue by iamnader** [react-native-community/react-native-side-menu#398]
# Question

1.1.3 is the latest release and is a little old

**Comment by Cookizza** [comment]
This component needs a few changes as it currently uses componentWillMount and componentWillReceiveProps - which are now deprecated.

**Comment by jdewells** [comment]
Would love to see a new release purely to fix the warnings for `componentWillReceiveProps`.

**Comment by cameronmurphy** [comment]
+1 here. This is the best alternative to `react-native-drawer` which has been abandoned. Even just a release that renames `componentWillMount` to `UNSAFE_componentWillMount` and `componentWillReceiveProps` to `UNSAFE_componentWillReceiveProps` would be handy so we can be compatible with React 17

**Comment by gonzalodiazcavuoti** [comment]
Any news about this?

**Comment by alessiocancian** [comment]
Hi guys, I released an updated version of this package to npm, you can find it here: https://www.npmjs.com/package/react-native-side-menu-updated

You can install it and change your imports from `react-native-side-menu` to `react-native-side-menu-updated` to use it.

I found that this repository already has all the changes about `UNSAFE_componentWillMount` and `UNSAFE_componentWillReceiveProps`, and the fixes for `Animated useNativeDriver not specified`, I added only `index.d.ts` and published the package.

**Comment by gonzalodiazcavuoti** [comment]
Thanks @alessiocancian . It worked for me

**Comment by dezudas** [comment]
@alessiocancian  thank you for the initiation, it could have been great if this was updated.

**Comment by jcarioti** [comment]
All these new commits mean nothing if there isn't a release! What's going on with regards to getting these changes pushed? It's been crickets in here for a while.

**Comment by danleveille** [comment]
@alessiocancian Is your updated package on Github too?

**Comment by safaiyeh** [comment]
cc @Naturalclar

**Comment by alessiocancian** [comment]
> @alessiocancian Is your updated package on Github too?

Yes, you can find the fork here: https://github.com/alessiocancian/react-native-side-menu

**Comment by safaiyeh** [comment]
Hey all! Apologies for the lack of releases, I'll break down the situation with this package and gauge how y'all feel about the future of it.

Current problem is that the org does not own the npm package react-native-side-menu and the owner is no longer actively maintaining the repo or making releases.

Our options:
1. The org no longer maintains react-native-side-menu and people are free to fork the repo and continue better maintenance as @alessiocancian took initiative on.
2. We release the package under the org `@react-native-community/side-menu` and @Naturalclar can help maintain and I'm sure someone from the community can join to help as well.

Let me know what y'all think!

**Comment by danleveille** [comment]
@safaiyeh Either option sounds great to me. As long as someone can approve pull requests, I'm sure either option will work. 😄

**Comment by GamesOfSummer** [comment]
Either works, just let us know the decision.

**Comment by safaiyeh** [comment]
Hey all,

Some news on the front. There will be some changes in the react-native-community and we won't be supporting as many modules. Would love to see an active maintained fork to succeed this.

We are going to go with option #1 I mentioned above and I'm willing help out the transition. Depending on if @alessiocancian wants to help support future development or a different fork, let me know!

Apologies for the lack of releases and this limbo state of nothing happening.
